src/utils.ts

```diff
@@ -2592,8 +2595,9 @@ export class newAxis extends EventEmitter {
   public normedValue(value: number): number { return value / this.interval }
 
   private computeMinMax(vector: any[]): number[] {
-    if (vector.length == 0) return [0, 1];;
-    if (this.isDiscrete) return [0, this.labels.length - 1]
+    if (!vector) return [0, 1];
```

Comment:
I think these two statements can be refactored to
```ts
if (!vector?.length) return [0, 1];
```

Isn't there any way for vector not to be undefined when isDiscrete should be true ?

Reply:
Nice !

When vector is defined, everything is under control. The problem appears when vector is undefined. This can only occurs when elements is empty or when specified attributes to plots are not in elements keys.

For the second case, this will raise an error before getting in this part of the program and is an error from user and not from plot_data. So I guess I can avoid considering it for the moment.

Reply:
I think that, at some point you'll have to handle it and set vector as null in these case. Something can be undefined for a whole lot of reasons so its better to fully control known cases by setting something to null, instead of handling undefined cases.
You could even raise an exception if error is on user's side